upgrading/extending-daml-models/2_new_package.md

```diff
@@ -0,0 +1,91 @@
+In this step, you'll create a new package depending on `create-daml-app` from scratch. We will be
+implementing the back-end for a discussion forum. The users will be identified by their user data
+from the mini social network of the Getting Started Guide. Thus we don't need to reimplement the
+user roles.
+
+In the terminal, change directory to your home directory and create a new empty DAML project:
+
+```
+cd
+daml new forum
+cd forum
+```{{execute T1}}
+
+The first thing you need to do is to add the `create-daml-app` to the dependencies in the
+`daml.yaml`{{open}} file. After, the file will look like this:
+
+<pre class="file" data-target="clipboard">
+sdk-version: 1.1.1
+name: forum
+source: daml
+parties:
+  - Alice
+  - Bob
+version: 0.0.1
+dependencies:
+  - daml-prim
+  - daml-stdlib
+  - daml-script
+  - ../create-daml-app/.daml/dist/create-daml-app-0.1.0.dar
+sandbox-options:
+  - --wall-clock-time
+</pre>
```

Comment:
This is also not gonna work as there's only one daml.yaml file in the IDE. I'm not 100% sure, but it might be that we'll need to add manually the second project to the IDE (or pre-load it for this step).

Reply:
I think it does, it's two separate directories. But I'll test again.

Reply:
I think this should work regardless because it only copies to the clipboard.